### PR TITLE
avoid astropy issues in last point release

### PR DESCRIPTION
**Pull request by ahnitz**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # requirements for most basic library use
 astropy>=2.0.3,<3.0.0; python_version <= '2.7'
-astropy>=2.0.3; python_version >= '3.0'
+astropy>=2.0.3!=4.2.1,!=4.0.5; python_version >= '3.0'
```
Comment (titodalcanton) [inline requirements.txt:3]: Is a comma needed between `2.0.3` and `!=`?

Comment (ahnitz) [reply requirements.txt]: Yes, see updated commit.

**Review by titodalcanton — APPROVED** [review]
Just for reference, this is a workaround for [this error](https://git.ligo.org/lscsoft/ligo.skymap/-/issues/26).

**Comment by ahnitz** [comment]
Here is the corresponding astropy report as well https://github.com/astropy/astropy/issues/11486

**Comment by titodalcanton** [comment]
@ahnitz the latest release of ligo.skymap includes a workaround for astropy 4.2.1. How do you feel about reverting this pin?

**Comment by ahnitz** [comment]
@titodalcanton It's not pinned, it just excludes the buggy astropy version. Newer version of astropy should be fine as they've fixed it on their end. Is there a reason we need to run 4.2.1?

**Comment by lpsinger** [comment]
What are the issues with astropy 4.2.1?

**Comment by ahnitz** [comment]
@lpsinger See the connected astropy issue. It has already been fixed upstream in astropy.

**Comment by lpsinger** [comment]
Well, as @titodalcanton said, there is also a workaround in ligo.skymap. So if you update to the latest version of ligo.skymap, then you can remove that astropy pin.

**Comment by ahnitz** [comment]
@lpsinger Is ligo.skymap pinned to 4.2.1? If so, then yep, go ahead and remove the exclusion to allow the latest ligo.skymap to be run for testing. I'm not sure when the next astropy release (with their fix) would be.

**Comment by titodalcanton** [comment]
I was asking because the latest version of ligo.skymap requires astropy 4.2.1 (in order to fix other bugs) and there has not been an astropy 4.2.2 in a while.

**Comment by ahnitz** [comment]
@titodalcanton Ok, I hadn't realized ligo.skymap was so restrictive. In which case, yeah, go ahead ,we weren't excluding 4.2.1 for any other reason I believe.

**Comment by lpsinger** [comment]
> @lpsinger Is ligo.skymap pinned to 4.2.1? If so, then yep, go ahead and remove the exclusion to allow the latest ligo.skymap to be run for testing. I'm not sure when the next astropy release (with their fix) would be.

https://git.ligo.org/lscsoft/ligo.skymap/-/blob/v0.5.3/setup.cfg#L42

**Pull request merged**